app-admin/logstash-bin/logstash-bin-2.3.3.ebuild

```diff
@@ -27,6 +28,10 @@ RDEPEND="|| ( virtual/jre:1.8 virtual/jre:1.7 )"
 S="${WORKDIR}/${MY_P}"
 
 pkg_setup() {
+	if use standard && use all-plugins; then
+		die "Both standard and all-plugins USE selected, please pick just one."
```

Comment:
Instead of doing it this way, look into having

```
REQUIRED_USE="?? ( standard all-plugins )"
```

This is a [at-most-one-of](https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-780008.2) dependency specification for USE flags